webview/src/experiments/components/table/body/PreviousCommitsRow.tsx

```diff
@@ -0,0 +1,22 @@
+import cx from 'classnames'
+import React from 'react'
+import styles from '../styles.module.scss'
+
+interface PreviousCommitsRowProps {
+  isBranchesView?: boolean
+  nbColumns: number
+}
+
+export const PreviousCommitsRow: React.FC<PreviousCommitsRowProps> = ({
+  isBranchesView,
+  nbColumns
+}) => (
+  <tbody>
```

Comment:
Same question for here as with the git branch row about this being in its own separate body.

Reply:
Just realized that this row is currently a `tbody` in `main` 🤔 Must have changed from a header row at some point :)